app/clients/insights_api_client.py

```
@@ -118,10 +111,10 @@ async def get_analytics_sentences(selected_area: dict, reference_area: dict) ->
         world_task = asyncio.create_task(
             query_insights_api(session, advanced_analytics_graphql)
         )
-        metadata_task = asyncio.create_task(
-            query_insights_api(session, indicators_graphql)
+        axes_task = asyncio.create_task(
+            query_insights_api(session, axis_graphql)
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)